lib/backend/etcdbk/etcd.go

```diff
@@ -23,7 +23,9 @@ import (
 	"crypto/tls"
 	"crypto/x509"
 	"encoding/base64"
+	"errors"
 	"io/ioutil"
+	"os"
```

Comment:
Why the os import?

Reply:
Thanks! Missed this while fixing conflicts. Fixed in a646441836b78cc06891456c70753ae1c5e0a5b4.